geometry/geometry_tests/geometry_tests.pro

```diff
@@ -7,7 +7,7 @@ TEMPLATE = app
 
 ROOT_DIR = ../..
 # TODO(AlexZ): latlon.hpp introduces indexer dependency for tests,
-# because of indexer/mercator.hpp. Either move latlon to indexer, or mercator to geometry.
+# because of geometry/mercator.hpp. Either move latlon to indexer, or mercator to geometry.
```

Comment:
Так же убери зависимость в omim.pro в строке
```
    geometry_tests.depends = 3party base geometry indexer

```

> On Dec 7, 2015, at 14:09, mgsergio <notifications@github.com> wrote:
> 
> In geometry/geometry_tests/geometry_tests.pro <https://github.com/mapsme/omim/pull/871#discussion_r46809322>:
> 
> > @@ -7,7 +7,7 @@ TEMPLATE = app
> >  
> >  ROOT_DIR = ../..
> >  # TODO(AlexZ): latlon.hpp introduces indexer dependency for tests,
> > -# because of indexer/mercator.hpp. Either move latlon to indexer, or mercator to geometry.
> > +# because of geometry/mercator.hpp. Either move latlon to indexer, or mercator to geometry.
> А вот зависимость не убрал.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/871/files#r46809322>.
>